packages/withRequired/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "@apollo/utils.withrequired",
+  "version": "1.0.0",
+  "description": "TypeScript utility type WithRequired",
+  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/apollographql/apollo-utils.git",
+    "directory": "withRequired/"
```

(this is still wrong in some places, likely one you modeled after)

```suggestion
    "directory": "packages/withRequired/"
```

---

(fixed all in https://github.com/apollographql/apollo-utils/commit/79c742d2c0d31dc0952de68457216b074f3fed22)

---

done